### PR TITLE
Adds ProxyCABundleTooLarge failure catcher

### PR DESCRIPTION
**Pull request by iamkirkbater**
Initial work to add this, started on a Sunday from an error I received over the weekend so I don't forget to come back to this during the normal work week.

### CODE REVIEW

**Comment by abutcher** [comment]
@iamkirkbater This looks fine to me, undraft when it is good to go

**Comment by 2uasimojo** [comment]
@iamkirkbater going to need to run `make update-gofmt` (or otherwise fix spacing in the .go file).

(I don't think the python syntax error is holding up the build, but it's being fixed in #1759 anyway.)

File: pkg/controller/clusterprovision/installlogmonitor_test.go [hunk 1]
```diff
@@ -141,6 +142,11 @@ func TestParseInstallLog(t *testing.T) {
 			log:            pointer.StringPtr(proxyInvalidCABundleLog),
 			expectedReason: "ProxyInvalidCABundle",
 		},
+		{
+			name:			"ProxyCABundleTooLarge",
+			log:			pointer.Stringptr(proxyCABundleTooLarge),
```
Comment (2uasimojo) [inline pkg/controller/clusterprovision/installlogmonitor_test.go:147]:
```suggestion
			name:		"ProxyCABundleTooLarge",
			log:		pointer.Stringptr(proxyCABundleTooLarge),
```

**Comment by 2uasimojo** [comment]
@iamkirkbater Tried to do this on the GH web UI by eyeball and obvs mucked it up.
Also, still seeing problems with the gitpython import. That's not what's causing the failure, but jsyk it's unrelated to this PR. I'm working on it separately.

**Pull request closed without merging**